Cannot find module './_export'

I have this with 0.8.0 and not with 0.7.0.
```
  /.../node_modules/mock-cloud-firestore/lib/mock-cloud-firestore.js:67

  Error: Cannot find module './_export'
```
I'm not sure if this is a problem on my end?

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -9,6 +9,7 @@ export default [
     output: { file: 'lib/mock-cloud-firestore.js', format: 'cjs' },
     plugins: [
       resolve(),
+      commonjs(),
       babel({
         exclude: 'node_modules/**',
       }),
@@ -21,6 +22,7 @@ export default [
     output: { file: 'es/mock-cloud-firestore.js', format: 'es' },
     plugins: [
       resolve(),
+      commonjs(),
       babel({
         exclude: 'node_modules/**',
       }),

